report_compassion/report/paperformats.xml

```diff
@@ -63,4 +63,20 @@
         <field name="header_spacing">3</field>
         <field name="dpi">96</field>
     </record>
+
+    <record id="paperformat_single_bvr_sponsorship" model="report.paperformat">
+        <field name="name">Single BVR Sponsorship</field>
+        <field name="default" eval="True"/>
+        <field name="format">custom</field>
+        <field name="orientation">Landscape</field>
+        <field name="page_height">205</field>
+        <field name="page_width">110</field>
+        <field name="margin_top">0</field>
+        <field name="margin_bottom">0</field>
+        <field name="margin_left">0</field>
+        <field name="margin_right">0</field>
+        <field name="header_line" eval="False"/>
+        <field name="header_spacing">3</field>
+        <field name="dpi">46</field>
```

Comment:
Normally we usde dpi 96 in order to get correct positioning in CSS with wkhtmltopdf.
```suggestion
        <field name="dpi">96</field>
```

Reply:
If we use the 96 dpi, the bill is not full page